photos.html

```diff
@@ -49,20 +49,22 @@ <h2 class="preamble__header preamble__header--small">Photo and video</h2>
 
       <div class="gallery">
         <section class="gallery__item">
-          <div class="gallery__inner">
+          <div class="gallery__inner gallery__inner--top">
             <div class="gallery__picture">
```

Comment:
Ok, I have a proposal. Let's make use of `figure` tag here for `gallery__picture` along with `figcaption` for `.gallery__caption` here's a doc https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure take a look at the example.
![image](https://user-images.githubusercontent.com/1570578/38166241-80c9565a-3518-11e8-93cd-6c070979c1d3.png)

 #### You will probably need to to the following:
1) change tags from `div` to `figure` and `figcaption`
2) move  `figcaption` inside `figure` tags next to pictures. (it makes more sense on a desktop as well)